pages/new-contribution-flow.js

```diff
@@ -180,8 +179,7 @@ class NewContributionFlowPage extends React.Component {
           collective={account}
           host={account.host}
           tier={tier}
-          feesOnTopAvailable={feesOnTopAvailable}
-          taxDeductible={taxDeductible}
+          isTaxDeductibleInTheUS={isTaxDeductibleInTheUS}
```

Comment:
Might be a small merge conflict here as I added some more props to this component but probably not too difficult to fix